src/patternfly/components/ProgressStepper/progress-stepper.scss

```
@@ -15,6 +15,7 @@
   --pf-c-progress-stepper--m-center__step-connector--JustifyContent: center;
 
   // Step icon variables
+  --pf-c-progress-stepper__step-icon--ZIndex: var(--pf-global--ZIndex--sm);
```

will this work as xs?
```suggestion
  --pf-c-progress-stepper__step-icon--ZIndex: var(--pf-global--ZIndex--xs);
```